src/components/main/section/search/SearchResultList.tsx

```diff
@@ -0,0 +1,35 @@
+import useFetchSearchResults from '../../../../hooks/useFetchSearchResults';
+import { SearchErrorItemStyled } from './SearchErrorItemStyled.styled';
+import SearchResultItem from './SearchResultItem';
+import { SearchResultListWrapper } from './SearchResultListWrapper.styled';
+
+interface SearchResultListProps {
+  searchTerm: string;
+}
+
+const SearchResultList = ({ searchTerm }: SearchResultListProps) => {
+  const { isLoading, error, data: topics } = useFetchSearchResults(searchTerm);
+
+  return (
+    <SearchResultListWrapper>
+      {error && <SearchErrorItemStyled>{error}</SearchErrorItemStyled>}
+      {topics && topics.length === 0 && (
+        <SearchErrorItemStyled>
+          Hledaný výraz není v naší databáze
```

Comment:
just typo

Reply:
like this? "Hledaný výraz není v naší databázi" ? :D Google translate helped me